x-pack/solutions/observability/plugins/slo/public/pages/slo_details/components/error_budget_chart.tsx

```diff
@@ -35,67 +36,70 @@ export interface Props {
   data: ChartData[];
   isLoading: boolean;
   slo: SLOWithSummaryResponse;
-  hideMetadata?: boolean;
   onBrushed?: (timeBounds: TimeBounds) => void;
 }
 
-export function ErrorBudgetChart({ data, isLoading, slo, hideMetadata = false, onBrushed }: Props) {
+export function ErrorBudgetChart({ data, isLoading, slo, onBrushed }: Props) {
   const { uiSettings } = useKibana().services;
   const percentFormat = uiSettings.get('format:percent:defaultPattern');
-  const isSloFailed = slo.summary.status === 'DEGRADING' || slo.summary.status === 'VIOLATED';
-  let errorBudgetTimeRemainingFormatted;
-  if (slo.budgetingMethod === 'timeslices' && slo.timeWindow.type === 'calendarAligned') {
-    const totalSlices =
-      toMinutes(toDuration(slo.timeWindow.duration)) /
-      toMinutes(toDuration(slo.objective.timesliceWindow!));
-    const errorBudgetRemainingInMinute =
-      slo.summary.errorBudget.remaining * (slo.summary.errorBudget.initial * totalSlices);
+  const isSloFailedStatus = isSloFailed(slo.summary.status);
+  const lastErrorBudgetRemaining = data.at(-1)?.value;
+
+  const errorBudgetTimeRemainingFormatted = (() => {
+    if (slo.budgetingMethod === 'timeslices' && slo.timeWindow.type === 'calendarAligned') {
+      const totalSlices =
+        toMinutes(toDuration(slo.timeWindow.duration)) /
+        toMinutes(toDuration(slo.objective.timesliceWindow!));
+      const errorBudgetRemainingInMinute =
+        slo.summary.errorBudget.remaining * (slo.summary.errorBudget.initial * totalSlices);
+
+      return formatTime(errorBudgetRemainingInMinute >= 0 ? errorBudgetRemainingInMinute : 0);
+    }
+    return undefined;
+  })();
+
+  const metadata = (
+    <EuiFlexGroup direction="row" gutterSize="l" alignItems="flexStart" responsive={false}>
+      <EuiFlexItem grow={false}>
+        <EuiStat
+          titleColor={isSloFailedStatus ? 'danger' : 'success'}
+          title={
+            lastErrorBudgetRemaining ? numeral(lastErrorBudgetRemaining).format(percentFormat) : '-'
+          }
+          titleSize="s"
+          description={i18n.translate('xpack.slo.sloDetails.errorBudgetChartPanel.remaining', {
+            defaultMessage: 'Remaining',
+          })}
+          reverse
+        />
+      </EuiFlexItem>
+      {errorBudgetTimeRemainingFormatted ? (
+        <EuiFlexItem grow={false}>
+          <EuiStat
+            titleColor={isSloFailedStatus ? 'danger' : 'success'}
+            title={errorBudgetTimeRemainingFormatted}
+            titleSize="s"
+            description={i18n.translate('xpack.slo.sloDetails.errorBudgetChartPanel.remaining', {
+              defaultMessage: 'Remaining',
+            })}
+            reverse
+          />
+        </EuiFlexItem>
+      ) : null}
+    </EuiFlexGroup>
+  );
```

Comment:
No need for extracting this into a variable since it's used only once in the return.